### PR TITLE
Update EKS_DISTRO_TAG_FILE.yaml

### PR DESCRIPTION
**Pull request by bhavi-koduru**
*Issue #, if available:*

*Description of changes:*
trigger a rebuild for the eks-distro-minimal-base-iptables to take the latest Golang changes

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: EKS_DISTRO_TAG_FILE.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ al2:
   eks-distro-minimal-base: 2023-09-06-1694026927.2
   eks-distro-minimal-base-nonroot: 2023-09-06-1694026927.2
   eks-distro-minimal-base-glibc: 2023-09-13-1694631680.2
-  eks-distro-minimal-base-iptables: 2023-09-13-1694631680.2
+  eks-distro-minimal-base-iptables: null
```
Comment (orsenthil) [inline EKS_DISTRO_TAG_FILE.yaml:6]: why is this set to null now?

Comment (rcrozean) [reply EKS_DISTRO_TAG_FILE.yaml:6]: Setting these values to null trigger automation to force rebuild the images. https://github.com/aws/eks-distro-build-tooling/pull/1166

**Pull request merged**